build(aio): upgrade CLI version to cope with new Angular 6.0.0-beta.1 release

petebacondarwin opened the pull request:
This should un-break master.

Before version 1.6 of Angular CLI there was a check that prevented use of Angular
compiler CLI with major version 6.

petebacondarwin committed:
build(aio): upgrade CLI version to cope with new Angular 6.0.0-beta.1 release

Before version 1.6 of Angular CLI there was a check that prevented use of Angular
compiler CLI with major version 6.

gkalpak reviewed (approved):
LGTM (as long as Travis is happy :smiley:)

mary-poppins commented:
You can preview cec3707 at https://pr21772-cec3707.ngbuilds.io/.

petebacondarwin committed:
docs(aio): fix paths to imported CSS stylesheets

The AOT compiler needs relative paths so that it can find
the imported stylesheets.

mary-poppins commented:
You can preview 569f718 at https://pr21772-569f718.ngbuilds.io/.

petebacondarwin committed:
docs(aio): fix missing stylesheet in component-styles example

The code in the example was referring to `hero-app.component.css` but this did
not exist.

petebacondarwin commented on aio/content/guide/component-styles.md:
```diff
@@ -177,8 +177,8 @@ to a component's `@Component` decorator:
   <code-pane title="src/app/hero-app.component.ts (CSS in file)" path="component-styles/src/app/hero-app.component.1.ts"></code-pane>
-  <code-pane title="src/app/hero-app.component.css" path="component-styles/src/app/hero-app.component.1.css"></code-pane>
-</code-tabs> 
+  <code-pane title="src/app/hero-app.component.css" path="component-styles/src/app/hero-app.component.css"></code-pane>
```
This is the only significant change in this file.
The rest is just trailing whitespace removal

gkalpak replied:
Why rename it? AFAICT, the file wan intentionally suffixed `.1.css`, so that it would not be included n stackblitz/zip.

petebacondarwin replied:
Because there is a component that references this file (without the 1) and breaks the build if the file is not there.

It is a trade off we could rename it in that component but then the 1 would leak into the docs.

mary-poppins commented:
You can preview 0e7b6d5 at https://pr21772-0e7b6d5.ngbuilds.io/.
